<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StitchWindow</class>
 <widget class="QDialog" name="StitchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StitchWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame{
/*background-color: rgb(125, 142, 255);*/
border-image:url(./background.jpg);
border-radius:30px;
padding:20px;
}
QPushButton{
	font: 75 italic 9pt &quot;Arial&quot;;
    border:0px;
    background:rgba(255,255,255,0);
	color: rgb(228, 243, 255);
}
QPushButton:hover                                               
{
    background: rgba(30,144,255,0.7);                         
}
QLabel
{
    color: rgb(228, 243, 255);   
}
QListView
{
	background-color: rgba(163, 189, 229,0.7);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>0</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minimizeButton">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>0</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>—</string>
      </property>
     </widget>
     <widget class="QListView" name="fileNameView">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>721</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>文件列表：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="featureSelect">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>99</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>surf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>orb</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>特征提取算法：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>warper算法：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cameraSelect">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>210</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>spherical</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cylindrical</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>210</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>合成时间(ms)：</string>
      </property>
     </widget>
     <widget class="QLabel" name="timeLabel">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>240</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(163, 189, 229,0.7);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="seamSelect">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>210</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>dp_colorgrad</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>gc_colorgrad</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>dp_color</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>gc_color</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>180</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>缝隙填补算法：</string>
      </property>
     </widget>
     <widget class="QLabel" name="result">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>390</y>
        <width>801</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="matchConf">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>280</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0.43</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>250</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>特征匹配阈值：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cuty">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>30</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cutx">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>270</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>纵向裁剪：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>250</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>横向裁剪：</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>80</y>
        <width>112</width>
        <height>117</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="openImageButton">
         <property name="text">
          <string>从图像中获取</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openVideoButton">
         <property name="text">
          <string>从视频中获取</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="outImageButton">
         <property name="text">
          <string>图像拼接</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="outVideoButton">
         <property name="text">
          <string>合成视频</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>180</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>捆集误差类型：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>250</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>延迟帧数:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="errorSelect">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>210</y>
        <width>99</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>reproj</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>310</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>featureScale:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="featureScale">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>330</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="composeScale">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>330</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>0.7</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="seamScale">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>330</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>310</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>ComposeScale:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>310</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>SeamScale:</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>280</y>
        <width>160</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QLineEdit" name="frameDelay_2">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="frameDelay">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="frameDelay_3">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="frameDelay_4">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
